src/test/java/org/jenkinsci/plugins/workflow/multibranch/SCMBinderTest.java

```diff
@@ -102,7 +102,7 @@
         WorkflowMultiBranchProjectTest.showIndexing(mp);
     }
 
-    static void assertRevisionAction(WorkflowRun build) {
+    public static void assertRevisionAction(WorkflowRun build) {
```

Comment:
Well, it is for tests only—if you know you need to use it, you can look at the source and find usages.